app/knight_arena/fight.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+
+from app.peoples.knight_create import KnightCreate
+
+
+class Fight:
+    def __init__(self, first_knight: "KnightCreate",
+                 second_knight: "KnightCreate") -> None:
+        self.first_knight = first_knight
+        self.second_knight = second_knight
+
+    def start_battle(self) -> tuple[Any, Any]:
+        damage_to_first_knight = (
+            max(self.second_knight.power - self.first_knight.protection, 0))
+        self.first_knight.hp -= damage_to_first_knight
+        self.fell_checkout()
+
+        damage_to_second_knight = (
+            max(self.first_knight.power - self.second_knight.protection, 0))
+        self.second_knight.hp -= damage_to_second_knight
+        self.fell_checkout()
```

Comment:
Consider refactoring the repetitive code for calculating damage to adhere to the DRY principle. You can create a helper method to handle the damage calculation for both knights.